store/postgres/src/store.rs

```diff
@@ -727,18 +749,44 @@ impl Store {
             }
         };
 
-        // Parse the schema and add @subgraphId directives
+        // Parse the schema
         let input_schema = Schema::parse(&input_schema, subgraph_id.clone())?;
-        let mut schema = input_schema.clone();
 
         // Generate an API schema for the subgraph and make sure all types in the
         // API schema have a @subgraphId directive as well
+        let (imported_schemas, schema_import_errors) = self.resolve_import_graph(&input_schema);
+        let mut unstable_schemas = imported_schemas
+            .iter()
+            .filter_map(|(schema_reference, schema)| match schema_reference {
+                SchemaReference::ByName(_) => {
+                    Some((schema_reference.clone(), Some(schema.id.clone())))
+                }
+                _ => None,
+            })
```

Comment:
Aren't references by ID also unstable, as they may or may not be available all the time?